test/test_compare.py

```diff
@@ -7,6 +7,11 @@
     ("str_expr", "str_expected", "fail_msg"),
     [
         # Equal (==)
+        (
+            "ClearAll[g,a,b];",
```

Comment:
I have looked at your old commit and seems you missed a (None, None, None) I guess that is why it wasn't working.